.github/workflows/ping-other-repos.yml

```
@@ -0,0 +1,19 @@
+name: Ping other repos
+
+on:
+  push:
+    branches: [main]
+  repository_dispatch:
+
+jobs:
+  ping-pong:
+    runs-on: ubuntu-latest
+    env:
+      SIDESHOWBARKER_TOKEN: ${{ secrets.SIDESHOWBARKER }}
+    if: ${{ github.respository == 'mdn/browser-compat-data' && env.SIDESHOWBARKER_TOKEN != '' }}
```

Oh, I see. It's probably unnecessary to do both guards, right? I mean, maybe there's a case to guard against unset secrets at the level of each step, if we wanted to add more dispatches. But let's cross that bridge when we get to it.

```suggestion
    if: ${{ github.respository == 'mdn/browser-compat-data' }}
```